src/components/products/filtersSection/filtersSection.tsx

```diff
@@ -0,0 +1,80 @@
+import { useSearchParams } from "react-router-dom";
+
+import SectionWrapper from "@/elements/sectionWrapper/sectionWrapper";
+import SelectFilterWrapper from "@/elements/selectFilterWrapper/selectFilterWrapper";
+import SelectControl from "@/elements/controls/select/select";
+import SelectWrapper from "@/elements/selectWrapper/selectWrapper";
+
+import RadioControl from "@/elements/controls/radio/radio";
+import RadioFilterWrapper from "@/elements/radioFilterWrapper/radioFilterWrapper";
+import styles from "./filtersSection.module.scss";
+
+interface FilterSectionProps {
+  handleCriteriaInputChange: (event: CustomEvent) => void;
+  handleTypeInputChange: (event: CustomEvent) => void;
+  handleGenreInputChange: (event: CustomEvent) => void;
+  handleAgeInputChange: (event: CustomEvent) => void;
+}
+
+export default function FiltersSection({
+  handleCriteriaInputChange,
+  handleTypeInputChange,
+  handleGenreInputChange,
+  handleAgeInputChange,
+}: FilterSectionProps) {
+  const [searchParams] = useSearchParams();
+  return (
+    <div className={styles.alignTextCenter}>
+      <SectionWrapper heading={searchParams.get("category")?.toUpperCase()}>
+        <SelectFilterWrapper heading="Sort">
+          <SelectWrapper heading="Criteria">
```

Comment:
SelectWrapper is wrong here. You should customize style for `<wup-select> strong` field instead because when I click on `Criteria` label it should move focus to related control